llvm/test/Feature/Repo/Inputs/remove_GO_ticketnode.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import re
+
+def filter_definition (line, GOList, filterStr):
+  if any(f in line for f in GOList):
```

Comment:
The previous version checked that the line started with "define". Is there a reason for that being dropped?

Reply:
Mistakenly dropped the "define", I have put back, thanks.